modules/cnv_networking_wizard_fields_web.adoc

```diff
@@ -19,6 +19,9 @@
 |NETWORK CONFIGURATION
 |List of available *NetworkAttachmentDefinition* objects.
 
+|BINDING METHOD
+|List of available binding methods. For pod networking, the possible values are `masquerade` and `bridge`. The `masquerade` method is not supported for non-default networks.
```

Comment:
It is very important **not** to allow `bridge` on the default network. It is spells only trouble.

For the default pod network, the only possible value is `masquerade`. For secondary networks the only possible method is `bridge`.